### PR TITLE
Created cucumber tests for the pipeline

### PR DESCRIPTION
**Pull request by jhmjam**
Created cucumber tests for the pipeline

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -10,6 +10,8 @@ markdown = "*"
 flake8 = "*"
 pylint = "*"
 bandit = "*"
+behave = "*"
+selenium = "*"
```
Comment (JasonBellONS) [inline Pipfile:14]: should these not be dev packages now?

**Review by JasonBellONS — COMMENTED** [review]
Looks good, one question noted around the lockfile

File: Pipfile (hunk 1 above)
Comment (jhmjam) [reply Pipfile:14]: ah yes good point done

**Review by JasonBellONS — APPROVED** [review]
Looks good to me

**Pull request merged**